### DISCUSSION TITLE
Using with apollo-server-lambda generates a different schema

### DISCUSSION THREAD

**Discussion by hisuwh** [MichalLytek/type-graphql#883] [Q&A]
I'm trying to convert my application from node/express to run in netlify lambda functions.  I have got it working other than the schema generated by type-graphql changes for some reason and this then breaks my application.  

With apollo-server-express:
```gql
type Mutation {
  addEvent(input: EventInput!): Event!
  updateEvent(update: EventInput!, id: ID!): Event!
  deleteEvent(id: String!): Boolean!
  ...
}

type Query {
  events(pagination: PaginationType, sort: EventSort, filter: EventFilter): [Event!]!
  ...
}

// rest of schema unchanged
```

With apollo-server-lambda:
```gql
type Mutation {
  addEvent: Event!
  updateEvent: Event!
  deleteEvent: Boolean!
  ...
}

type Query {
  events: [Event!]!
  ...
}

// rest of schema unchanged
```

As you can see my mutation and query arguments are missing.  I'll try and post a minimal reproduction later but thought someone might have seen this before or know how to fix this?

**Comment by MichalLytek** [comment]
Invalid babel/webpack configuration which results in missing transpiled decorators for args.

**Reply by hisuwh** [reply]
I'm not using babel or webpack - I'm compiling with tsc

**Reply by MichalLytek** [reply]
But the serverless/lambda is bundling the code.
https://github.com/MichalLytek/type-graphql/issues/162
https://github.com/MichalLytek/type-graphql/issues/367

**Reply by hisuwh** [reply]
Looks like something strange is going on so in my build output I've got this for example:
```js
__decorate([
    type_graphql_1.Authorized(),
    type_graphql_1.Query(returns => WorkoutTotals_1.WorkoutTotals),
    __param(0, type_graphql_1.Arg("filter", type => Event_1.EventFilter, { nullable: true })),
    __metadata("design:type", Function),
    __metadata("design:paramtypes", [Event_1.EventFilter]),
    __metadata("design:returntype", void 0)
], EventResolver.prototype, "workoutTotals", null);
```
But in the bundled file under `.netlify/function-serve/graphql/src` this same function is output like this
```js
__decorateClass([
  (0, import_type_graphql11.Authorized)(),
  (0, import_type_graphql11.Query)((returns) => WorkoutTotals),
  __decorateParam(0, (0, import_type_graphql11.Arg)("filter", (type) => EventFilter, {nullable: true}))
], EventResolver.prototype, "workoutTotals", 1);
```

So it seems to be stripping out the metadata for some reason.  Though I can see the __metadata function being defined in the bundle.  I believe this is the tool netlify uses to bundle it up though which I don't think does anything clever:
https://github.com/netlify/zip-it-and-ship-it

**Reply by hisuwh** [reply]
Mmm so digging into this more, despite the netlify docs saying they bundle with the mentioned packaged, it is not a dependency of their CLI and their CLI seems to take a dependency on babel - dirty
